www/includes/easyparliament/templates/html/mp/_vote_description.php

```diff
@@ -7,5 +7,8 @@
         <?= isset($key_vote) ? "$key_vote[summary]." : '' ?>
         <?= isset($party_voting_line) ? $party_voting_line : '' ?>
     </a>
+    <?php if (isset($covid_affected) && $covid_affected) { ?>
```

Comment:
This template is included on profile.php as well as votes.php - just checking you only wanted it to be displayed on the votes.php entries.

Reply:
Ah right, these might appear in the summary items. Will add those in.